examples/transformer/user_script.py

```diff
@@ -0,0 +1,25 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import sys
+from pathlib import Path
+
+import torch
+
+wd = Path(__file__).parent.parent.parent.parent.resolve() / "orchard"
```

Comment:
this path is specific to your work environment and not generalizable. 

I think it is better to give instructions in a README to tell where to clone to orchard repository from. And to set an environment variable or something to point to the cloned repo.
Ideally, I think orchard should be pip installable python package so that we can just import the package directly. But I see that the orchard repository currently doesn't have `__init__.py` files and `setup.py`.